properties/color/brand.json

```
@@ -2,12 +2,12 @@
   "color": {
     "brand": {
       "primary": {
-        "base":    { "value": "{color.base.palatinate.900.value}" }
+        "base":    { "value": "{color.snow.light.name}" }
```

this color doesn't exists, you should change to a temporary color like this: 
```suggestion
        "base":    { "value": "{color.base.strawberry}" }
```

---

Me cargo todo el fichero de brand hasta que tengamos el branding (i.e. una manera de usar la paleta)